api/src/test/java/uk/gov/hmcts/payment/api/componenttests/PaymentControllerTest.java

```diff
@@ -380,10 +380,12 @@ public void searchAllPayments_withCcdCaseNumber_shouldReturnRequiredFieldsForVis
 
         assertThat(payment.getCcdCaseNumber()).isEqualTo("ccdCaseNumber1");
 
-        assertThat(payment.getReference()).isNotBlank();
+        assertThat(payment.getChannel()).isNotBlank();
         assertThat(payment.getAmount()).isPositive();
         assertThat(payment.getDateCreated()).isNotNull();
-        assertThat(payment.getCustomerReference()).isNotBlank();
```

Comment:
Please try to keep the same test case logic with new Library code

Reply:
Reverted to old library version